plugins/org.eclipse.thym.ui/src/org/eclipse/thym/ui/config/internal/PropertiesPage.java

```diff
@@ -298,15 +299,15 @@ public void selectionChanged(SelectionChangedEvent event) {
 					featureParamsTableViewer.setInput(null);
 					return;
 				}
-				Feature feature = (Feature)sel.getFirstElement();
-				featureParamsTableViewer.setInput(feature.getParams());
-				feature.addPropertyChangeListener("param", new PropertyChangeListener() {
-
-					@Override
-					public void propertyChange(PropertyChangeEvent event) {
-						featureParamsTableViewer.setInput(event.getNewValue());
-					}
-				});
+//				Feature feature = (Feature)sel.getFirstElement();
```

Comment:
These lines are commented out because <Plugin> tags do not have parameters. I wasn't sure if we wanted to remove the parameters section / the whole page altogether, so I've left it in for now.